### ISSUE TITLE
 download validates package

### ISSUE THREAD

**Issue by cweiske** [pyrus/Pyrus#20]
Installing a pecl package does not work because of an bug in it. Now I just wanted to download the package and fix it, but even when just downloading a package it validates the archive:

```
$ pyrus-5.3.5 download pecl/xdebug
Pyrus version 2.0.0a2 SHA-1: CA3946A5C2C7DE7CD6ED89CAC7CCC06606E006D3
Using PEAR installation found at /opt/phpfarm/inst/php-5.3.5/pear
Downloading pecl/xdebug...
Connected...
Mime-type: application/octet-stream

failed! (Invalid package downloaded, package name changed from Xdebug to xdebug)=====================>] 100% (282/282 kb)
```
## Expected result:

I want a downloaded file that I can fix. Pyrus should not verify it.

**Comment by saltybeagle** [comment]
I can't reproduce this. I tried downloading 1.3.0 - 2.1.3 and all versions were able to download. What version were you trying to download?

**Comment by cweiske** [comment]
Cannot reproduce anymore with 2.0.0a3. Probably fixed since my 2.0.0a2
